building/gits/build_pre_v3.rst

```diff
@@ -0,0 +1,55 @@
+.. _build_pre_v3:
```

Comment:
I would simply call it `build_legacy.rst`.

Reply:
Also note that you need to add this new file to the index file (`building/index.rst`). That is the reason why you have the error in Travis.

Reply:
ok, thanks.